### DISCUSSION TITLE
The USB network card is cf-811ac, and the V12 driver has been installed, but the USB WiFi is not displayed in the network，sip is disabled

### DISCUSSION THREAD

**Discussion by guchenghua** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#184] [Q&A]
The USB network card is cf-811ac, and the V12 driver has been installed, but the USB WiFi is not displayed in the network，sip is disabled

**Comment by chris1111** [comment]
The discussion looks[ like an issue](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/discussions/19)
